src/list/list.css

```diff
@@ -60,9 +58,9 @@ nav {
     display: block;
 }
 
-.exploreprovider {
-    position: sticky;
-    top: var(--topbar-height);
+.scrollable {
+    max-height: 500px;
```

Comment:
I think this should be something like `calc(600px - var(--topbar-height))`.

Reply:
It's not be enough when the header also includes the provider's selector and/or the search field.

Reply:
Right, so those should increase the var or have their own additional vars then.